.github/workflows/ci.yml

```diff
@@ -2,13 +2,9 @@ name: CI
 
 on:
   workflow_dispatch:
-    inputs:
-      run_integration_tests:
```

Comment:
This wasn't being used at all, it was introduced in a previous PR with the attempt to continue using `pull_request` event with [workflow_run](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_run)